Libraries/CustomComponents/NavigationExperimental/NavigationCardStackPanResponder.js

```diff
@@ -45,7 +45,7 @@ const RESPOND_THRESHOLD = 15;
  * standard maximum position to start touch responsiveness.
  */
 const RESPOND_POSITION_MAX_HORIZONTAL = 30;
-const RESPOND_POSITION_MAX_VERTICAL = null;
+const RESPOND_POSITION_MAX_VERTICAL = 64;
```

Comment:
This is different on Android, and also differs when the status bar is hidden on iOS. (And the nav bar height might differ when the phone is in landscape mode.) Basically rather than using a magical constant that happens to work today but could break tomorrow, I think we should look for touches on the nav bar header or measure its dimensions and reduce content coupling.